rukundoeric/bookstore
# Milestone 2: Implement components

## What does this do? 
*  Implement components
*  Connect components to redux store
## Description of Task to be completed? 
*  In this milestone, I implemented components and connected some of them to redux store.
## How should this be manually tested? 
NOTE: Make sure you have Node.js and npm or yarn installed in your PC.
- Clone this repo: `git clone https://github.com/rukundoeric/bookstore.git`
- Navigate into project directory: `cd bookstore`
- Install all packages: `yarn install or npm install`
- Start the app: `yarn start or npm start`
